src/device/device.cpp

```diff
@@ -27,6 +27,8 @@
 // THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 
+#include <thread>
```

Comment:
This is in a cpp, and no other change. Spurious ?

Reply:
fixed